uw-frame-java/pom.xml

```diff
@@ -218,43 +180,277 @@
+            <id>copy-node_modules</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/frame/css/themes/node_modules/</outputDirectory>
+              <resources>          
+                <resource>
+                  <directory>${basedir}/../node_modules/</directory>
+                  <include>normalize.less/</include>
+                  <include>bootstrap/</include>
+                  <include>font-awesome/</include>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+              <nonFilteredFileExtensions>
+                <nonFilteredFileExtension>woff</nonFilteredFileExtension>
+                <nonFilteredFileExtension>woff2</nonFilteredFileExtension>
+                <nonFilteredFileExtension>ttf</nonFilteredFileExtension>
+                <nonFilteredFileExtension>eot</nonFilteredFileExtension>
+                <nonFilteredFileExtension>otf</nonFilteredFileExtension>
+              </nonFilteredFileExtensions>          
+            </configuration>            
+          </execution>
+          <execution>
+            <id>copy-static-feeds</id>
+            <!-- here the phase you need -->
+            <phase>validate</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/frame/staticFeeds</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/main/filtered-resources</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-css</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/frame/css</outputDirectory>
+              <resources>          
+                <resource>
+                  <directory>${basedir}/../uw-frame-static/target/css</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>              
+            </configuration>            
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>com.coderplus.maven.plugins</groupId>
+        <artifactId>copy-rename-maven-plugin</artifactId>
+        <version>1.0.1</version>
+        <executions>
+          <execution>
+            <id>rename-uw-madison-file</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>rename</goal>
+            </goals>
+            <configuration>
+              <sourceFile>${basedir}/target/frame/css/themes/uw-madison.css</sourceFile>
```

Comment:
I admittedly tried for a while and wasn't able to do it.  MUMUP-2810 to track this.  This could also be a step in the right direction to allow us to be a little bit less UW specific.